app/assets/stylesheets/modules/info-window.css.scss

```diff
@@ -48,3 +48,14 @@
   font-weight: $font-stack-bold;
   margin-top: 10px;
 }
+
+.info_window__avg-color {
+  min-width: 9px;
+  min-height: 9px;
+  max-width: 9px;
+  max-height: 9px;
```

Comment:
why min and max instead of just width & height?

Reply:
That's how it's done for a similar dot in `single-session.css.scss` in `.single-session__avg-color `. Do you think it's not needed here?

Reply:
ok, don't worry, I'll take a look. looks weird! maybe some old code

Reply:
✅